src/backend/distributed/transaction/worker_transaction.c

```diff
@@ -340,6 +343,36 @@ void
 SendCommandToWorkersOptionalInParallel(TargetWorkerSet targetWorkerSet, const
 									   char *command,
 									   const char *user)
+{
+	bool isOptionalSend = true;
+	SendCommandToWorkersOutsideTransaction(targetWorkerSet, command, user,
+										   isOptionalSend);
+}
+
+
+/*
+ * SendCommandToWorkersInParallel sends the given command to workers in parallel.
+ * It does error if there is a problem while sending the query, it errors if there
+ * was any problem when sending/receiving.
+ */
+void
+SendCommandToWorkersInParallel(TargetWorkerSet targetWorkerSet, const
+							   char *command,
+							   const char *user)
+{
+	bool isOptionalSend = false;
+	SendCommandToWorkersOutsideTransaction(targetWorkerSet, command, user,
+										   isOptionalSend);
```

Comment:
maybe isOptionalSend -> failOnError (and invert the meaning), since we've used that elsewhere in this file.